persist/fs/types.go

```diff
@@ -67,6 +67,10 @@ type FileSetReader interface {
 	// Use either Read or ReadMetadata to progress through a volume, but not both.
 	ReadMetadata() (id ts.ID, length int, checksum uint32, err error)
 
+	// ReadBloomFilter returns the bloom filter stored on disk in a container object that is safe
+	// for concurrent use and has a Close() method for releasing resources when done.
+	ReadBloomFilter() (managedConcurrentBloomFilter, error)
```

Comment:
This method won't be useful outside of this package if we don't return an exported type.

Reply:
same comment as above